tdrs-frontend/package.json

```diff
@@ -119,5 +119,13 @@
       "cypress/e2e/accounts/*.js",
       "cypress/e2e/integration/*.js"
     ]
+  },
+  "overrides": {
+    "semver": "^7.5.1",
+    "nth-check": "^2.0.1",
+    "braces": "^3.0.3",
+    "tough-cookie": "^4.1.3",
+    "postcss": "^8.4.31",
+    "micromatch": "^4.0.8"
```

Comment:
will these have to be changed/removed when the downstream packages update their dependencies?

Reply:
Yes, this is just interim solution